get-started/setup-lightdash/connect-project.mdx

```diff
@@ -891,7 +905,34 @@ If you've [customized the domain for your Bitbucket server](https://confluence.a
 
 ***
 
-### CLI 
+### dbt Cloud
+
+The dbt Cloud connection type bypasses any git connection, instead directly accessing metadata from your dbt Cloud project using the dbt Cloud API.
+
+This means a number of features are **not** available, including:
+* [dbt write-back features](/references/dbt-write-back)
+* Preview environments from git branches
```

Comment:
maybe add a link here: https://docs.lightdash.com/references/preview-projects#lightdash-app

and clarify "Preview environments created in the UI from git branches"